connectors/src/api/bot_enabled.ts

```diff
@@ -0,0 +1,134 @@
+import { Request, Response } from "express";
+
+import { Connector, SlackConfiguration } from "@connectors/lib/models";
+import { apiError, withLogging } from "@connectors/logger/withlogging";
+
+type GetSlackbotEnabledRes = {
+  botEnabled: boolean;
+};
+
+type ToggleSlackbotReqBody = {
+  botEnabled: boolean;
+};
+
+const _getBotEnabled = async (
+  req: Request<{ connector_id: string }, GetSlackbotEnabledRes, undefined>,
+  res: Response<GetSlackbotEnabledRes>
+) => {
+  if (!req.params.connector_id) {
+    return apiError(req, res, {
+      api_error: {
+        type: "invalid_request_error",
+        message: "Missing required parameters. Required: connector_id",
+      },
+      status_code: 400,
+    });
+  }
+  const connector = await Connector.findByPk(req.params.connector_id);
+  if (!connector) {
+    return apiError(req, res, {
+      api_error: {
+        type: "connector_not_found",
+        message: "Connector not found",
+      },
+      status_code: 404,
+    });
+  }
+  if (connector.type !== "slack") {
+    return apiError(req, res, {
+      api_error: {
+        type: "invalid_request_error",
+        message: "Connector is not a slack connector",
+      },
+      status_code: 500,
+    });
+  }
+
+  const slackConfig = await SlackConfiguration.findOne({
```

Comment:
This should live below the slack connector. We should add a switch similar to other endpoints here erroring if the connector does not support the bot commands?